mappings/net/minecraft/client/gui/screen/OrderedTextTooltip.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_5683 net/minecraft/client/gui/screen/OrderedTextTooltip
```

Comment:
Should this also have `Component` as a suffix like `BundleTooltipComponent`?

Reply:
Yep, that was a typo, it's been adjusted